app/views/JrsClaimSummaryView.scala.html

```diff
@@ -22,73 +22,79 @@
 @import scala.concurrent.ExecutionContext
 @import uk.gov.hmrc.tai.model.YearAndMonth.firstClaimDate
 @import includes.link
+@import includes.ptaHeader
 
-@this(mainTemplate: MainTemplate, languageUtils: LanguageUtils)
+@this(main: MainTempleteNew, languageUtils: LanguageUtils)
 
 @(viewModel: JrsClaims, appConfig: ApplicationConfig)(implicit request: Request[_], messages: Messages, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer,  ec: ExecutionContext)
 
 @employerMessageKey = @{viewModel.employerMessageKey}
 
-@mainTemplate(
-    title = messages("check.jrs.claims.title")
+
+@main(
+    title = messages("check.jrs.claims.title"),
+    pageTitle = Some(Messages("tai.service.navTitle")),
+    backLinkUrl = None,
```

Comment:
What happens when the user clicks to go back if we havent got a URL?

Reply:
This page is unique in the case that the back link is handled differently, on lines 48 - 55. Whether we want to handle this in a uniform way, that's up for discussion; however, I did not want to make changes which could affect the layout.
The user navigates back to the whatDoYouWantToDoPage.

So there is a back button, it's just not handled by the main template :)

Reply:
how come we cant have that back link in here? e.g. 
backLinkUrl = routes.WhatDoYouWantToDoController.whatDoYouWantToDoPage.url,
backLinkContent = Messages("check.jrs.back.link")

Reply:
Spoken with Vivek, opted to use the backLink generated from the mainTemplate. 

removed the HTML within the View, as both of them would generate two back links on the page.